src/modules/75-cf/strings/strings.en.yaml

```
@@ -131,6 +130,9 @@ featureFlags:
   newFlag: New Flag
   true: 'True'
   false: 'False'
+  noFlagsInProject: There are no Flags in this Project
+  noFlagsDescription: A Feature Flag controls what you want to release, who receives it, and how it will be delivered.
+  noFlagsToGetStarted: To get started with Feature Flags, you can <a href="https://docs.harness.io/article/1j7pdkqh7j-create-a-feature-flag"> create your first flag</a>, <a href="https://docs.harness.io/category/rtce97j1wu-ff-sdks"> get started with our SDKs</a>, or <a href="https://docs.harness.io/category/iqgyied1c0-ff-getting-started">read our getting started guide</a>.
```

One missed capitalisation and a couple of errant spaces
```suggestion
  noFlagsToGetStarted: To get started with Feature Flags, you can <a href="https://docs.harness.io/article/1j7pdkqh7j-create-a-feature-flag">create your first Flag</a>, <a href="https://docs.harness.io/category/rtce97j1wu-ff-sdks">get started with our SDKs</a>, or <a href="https://docs.harness.io/category/iqgyied1c0-ff-getting-started">read our getting started guide</a>.
```

---

Great spot, thank you - have fixed